File: problemas/ad-hoc/2375/2375.r
input <- file('stdin', 'r')

diametro <- as.integer(readLines(input, n=1))

entrada <- as.integer(strsplit(readLines(input, n = 1), ' ')[[1]])
altura <- entrada[1]
largura <- entrada[2]
profundidade <- entrada[3]

if (diametro <= altura && diametro <= largura && diametro <= profundidade) {
    write(paste('S'), '')
} else {
    write(paste('N'), '')
}

File: problemas/ad-hoc/2375/2375.swift
import Foundation

let diametro = Int(readLine(strippingNewline: true)!)!

var entrada = readLine(strippingNewline: true)!.components(separatedBy: " ")
let altura = Int(entrada[0])!
let largura = Int(entrada[1])!
let profundidade = Int(entrada[2])!

if (diametro <= altura && diametro <= largura && diametro <= profundidade) {
    print("S")
} else {
    print("N")
}